Runtime/ui/renderer/common/geometry/path/path_cache.cs

```diff
@@ -14,13 +14,24 @@ class uiPathCache : PoolObject {
 
         //mesh cache
         uiMeshMesh _fillMesh;
+
```

Comment:
Please make sure that ui/painting/path doesn't require the implementation of all these stuffs ? If they do we should modify it too